src/chains-config/metadata-consts/kusama-consts.ts

```diff
@@ -0,0 +1,62 @@
+import { Definition } from '../../types/chains-config';
+
+/**
+ * All Runtimes before v2027
+ */
+const extrinsicBaseWeight = 125000000;
+
+/**
+ * Runtimes after v2026
+ */
+const blockWeights = {
+	baseBlock: 5000000000,
+	maxBlock: 2000000000000,
+	perClass: {
+		normal: {
+			baseExtrinsic: 125000000,
+			maxExtrinsic: 371168000000001,
+			maxTotal: null,
+			reserved: 1500000000000,
+		},
+		operational: {
+			baseExtrinsic: 1,
+			maxExtrinsic: 32000000000,
+			maxTotal: null,
+			reserved: 1949875000000,
```

Comment:
Do we need all these fields? Maybe just `baseExtrinsic` in each perClass?

Reply:
Fixed!